src/app/utils/common.util.ts

```diff
@@ -30,34 +30,55 @@ export class CommonUtil {
     return uniqid;
   }
 
-  static formatBytes(value: number | string): string {
-    const units: readonly string[] = ['kB', 'MB', 'GB', 'TB', 'PB'];
+  static formatMemoryBytes(value: number | string): string {
+    const units: readonly string[] = ['KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB'];
     let unit: string = 'bytes';
-    let toValue = +value
+    let toValue = +value;
+    for (let i = 0, unitslen = units.length; toValue / 1024 >= 1 && i < unitslen;i = i + 1) {
+      toValue = toValue / 1024;
+      unit = units[i];
+    }
+    return `${toValue.toLocaleString(undefined, { maximumFractionDigits: 2 })} ${unit}`;
+  }
+
+  static formatEphemeralStorageBytes(value: number | string): string {
+    const units: readonly string[] = ['KB', 'MB', 'GB', 'TB', 'PB', 'EB'];
+    let unit: string = 'bytes';
```

Comment:
This should be just a `B` for consistency with the other units.

Reply:
Changed 'bytes' to 'B'